mail/rspamd/src/opnsense/scripts/rspamd/setup.sh

```diff
@@ -11,3 +11,8 @@ chmod o+rx /usr/local/etc/rspamd/local.d
 chown -R rspamd:rspamd /var/db/rspamd
 chown -R rspamd:rspamd /var/log/rspamd
 chown -R rspamd:rspamd /var/run/rspamd
+
+# Add 'rspamd' user to 'redis' group to allow Unix socket access
+if [ -f "/usr/local/bin/rspamd" ]; then
```

Comment:
the check is stupid - if you are in the rspamd plugin, rspamd is always installed.
I guess you wanted to check for the redis binary?